pyflakes/test/test_doctests.py

```diff
@@ -367,12 +367,7 @@ class TestOther(_DoctestMixin, TestOther):
 
 
 class TestImports(_DoctestMixin, TestImports):
-
-    def test_futureImport(self):
-        """XXX This test can't work in a doctest"""
-
-    def test_futureImportUsed(self):
-        """XXX This test can't work in a doctest"""
+    pass
```

Comment:
Should this class (and while we're at it, maybe the one above) just be removed? Unless I'm missing something, not much point to an empty test class.